src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -17,30 +27,160 @@ export const GRTSystemIntakeReview = () => {
+                          <HelpText>
+                            If there isn&apos;t enough info on this request,
+                            please get in touch with the requester over email
+                          </HelpText>
+                        </div>
+                        <div className="grid-col-2" />
+                        <div className="grid-col-6">
+                          <FieldGroup scrollElement="emailText">
+                            <Label
+                              className="margin-bottom-2 margin-top-0"
+                              htmlFor="GrtIntakeReviewForm-EmailText"
+                            >
+                              This email will be sent to the requester
+                            </Label>
+                            <Field
+                              as={TextAreaField}
+                              id="GrtIntakeReviewForm-GrtReviewEmailBody"
+                              maxLength={2000}
+                              name="grtReviewEmailBody"
+                              aria-describedby="GrtIntakeReviewForm-GrtReviewEmailBody"
+                              className="maxw-full"
+                            />
+                          </FieldGroup>
+                          <Button className="margin-top-3" type="submit">
+                            Email Decision and Progress to next step
+                          </Button>
+                        </div>
+                      </div>
+                    </Form>
+                  );
+                }}
+              </Formik>
+            )}
+            {['APPROVED', 'ACCEPTED', 'CLOSED'].includes(
+              systemIntake.status
+            ) && (
+              <div>
+                <h1 className="font-heading-xl margin-top-0">Next steps</h1>
+                <Alert type="success" heading="Email sent">
+                  An email has been sent to {systemIntake.requesterEmailAddress}
+                </Alert>
+                <p className="margin-bottom-3">
+                  Decision:{' '}
+                  {((status: SystemIntakeStatus) => {
+                    if (status === 'APPROVED') {
+                      return 'Issue Lifecycle ID with no further governance';
+                    }
+                    if (status === 'ACCEPTED') {
+                      return 'Governance review process needed';
+                    }
+                    if (status === 'CLOSED') {
+                      return 'Governance not needed (close this request)';
+                    }
+                    return '';
+                  })(systemIntake.status)}
+                </p>
+                <p className="text-pre-wrap line-height-body-3 margin-bottom-4">
+                  {systemIntake.grtReviewEmailBody}
+                </p>
+                <i>
+                  An email was sent to the requester on{' '}
+                  {systemIntake.reviewedAt &&
```

Comment:
If `systemIntake.reviewedAt` is falsy, it will display `An email was sent to the requester on `. In the (rare) case there's not a date, I'm not sure if we should hide that sentence completely or leave it as is.